php/commands/site.php

```diff
@@ -749,34 +739,35 @@ public function delete_env($args, $assoc_args) {
     */
    public function deploy($args, $assoc_args) {
      $site = SiteFactory::instance( Input::site( $assoc_args ) );
-     $env = Input::env($assoc_args);
+     $env  = $site->environment(Input::env(
+       $assoc_args,
+       'env',
+       'Choose environment you want to initialize',
+       array('test', 'live')
+     ));
      $from = Input::env($assoc_args, 'from', "Choose environment you want to deploy from");
```

Comment:
I think we should remove this line. The logic should be hardcoded:

- If you are deploying the **Test** environment, you deploy from the **Dev** environment, and you have the option to copy content and database from the **Live** environment 
- If you are deploying the **Live** environment, you deploy from the **Test** environment (and you do NOT give the option to copy anything over.